core/tests/unit/ldml/test_transforms.cpp

```diff
@@ -1033,6 +1033,44 @@ test_normalize() {
   return EXIT_SUCCESS;
 }
 
+#include <unicode/uchar.h>
+
+#define NFD_FILE "common/web/types/src/ldml-keyboard/nfd-table.ts"
+int
+write_nfd_table() {
```

Comment:
This function does not appear to be called from `main()`?

Reply:
bad merge, ok i will fix this.  (I had shelved this in favor of the segment)